tests/cypress/integration/actions_projects_models_organizations/registration_involved/case_113_new_organization_pipeline.js

```diff
@@ -0,0 +1,284 @@
+            cy.login();
+            cy.activateOrganization(organizationParams.shortName);
+            cy.openOrganization(organizationParams.shortName);
+            cy.contains('button', 'Leave organization').should('be.visible').click();
+            cy.get('.cvat-modal-organization-leave-confirm')
+                .should('be.visible')
+                .within(() => {
+                    cy.contains('button', 'Leave').click();
+                });
+            cy.get('.cvat-notification-notice-leave-organization-failed').should('exist');
+            cy.closeNotification('.cvat-notification-notice-leave-organization-failed');
+        });
+
+        it('Second user login. The user is able to see the organization, the task and can’t see the project.', () => {
+            cy.logout();
+            cy.login(secondUserName, secondUser.password);
+            cy.checkPresenceOrganization(organizationParams.shortName);
+            cy.contains('.cvat-item-task-name', taskName).should('exist');
+            cy.goToProjectsList();
+            cy.get('.cvat-empty-projects-list').should('exist');
+        });
+
+        it('Open the task, assign one of jobs to the third user. Rename the task.', () => {
+            cy.activateOrganization(organizationParams.shortName);
+            cy.goToTaskList();
+            cy.openTask(taskName);
+            cy.assignJobToUser(0, thirdUserName);
+            cy.renameTask(taskName, newTaskName);
+            cy.url().then((url) => {
+                taskID = Number(url.split('/').slice(-1)[0]);
+            });
+            cy.getJobNum(0).then(($jobID) => {
+                jobID = $jobID;
+            });
+        });
+
+        it('Logout, the third user login. The user does not see the project, the task. The user can open the job using direct link. Create an object, save annotations.', () => {
+            cy.logout(secondUserName);
+            cy.login(thirdUserName, thirdUser.password);
+            cy.contains('.cvat-item-task-name', taskName).should('not.exist');
+            cy.goToProjectsList();
+            cy.contains('.cvat-projects-project-item-title', project.name).should('not.exist');
+            cy.visit(`/tasks/${taskID}/jobs/${jobID}`);
+            cy.get('.cvat-canvas-container').should('exist');
+            cy.createCuboid(createCuboidShape2Points);
+            cy.saveJob();
+        });
+
+        it('The owner of the organization removes the second user from it.', () => {
+            cy.logout(thirdUserName);
+            cy.login(firstUserName, firstUser.password);
+            cy.activateOrganization(organizationParams.shortName);
+            cy.openOrganization(organizationParams.shortName);
+            cy.removeMemberFromOrganization(secondUserName);
+            cy.checkOrganizationMembers(2, [firstUserName, thirdUserName]);
+        });
+
+        it('The organization, project is no longer available to the second user. The task is available.', () => {
```

Comment:
Actually I believe it is a bug, I do not expect user out of organization can see the organization resources.

Reply:
I'll check

Reply:
OK

Reply:
Yes, it is a something like a bug, but we decided to keep it as is for now.